index.js

```diff
@@ -62,6 +68,8 @@ module.exports = {
             invalid_form: 'Dữ liệu nhập không đúng. Vui lòng kiểm tra lại',
             delete_title: 'Xóa %{name} #%{id}',
             delete_content: 'Bạn có chắc chắn muốn xóa vật phẩm này?',
+            details: 'Chi tiết',
+            error: 'Xảy ra lỗi máy khách và yêu cầu của bạn không thể hoàn thành',
```

Comment:
I think I will translate 'Đã xảy ra lỗi ở phía người dùng và yêu cầu của bạn không thể hoàn thành.' for this.